File: python/intro.py
def intro():
    x = 100
    y = 1.0

    print(f"x: {x}, and y: {y}")
    y = y * -0.3145
    print(f"x: {x}, and y: {y}")

    if y < x:
        print(f"The difference is: {x-y}")
    elif y == x:
        print(f"The difference is: {x-y}")
    else:
        print(f"The difference is: {y-x}")

    # x = 5
    # while x > 0:
    #     print(f"x is {x}")
    #     x -= 1

    for i in range(3, 7):
        print(f"i is {i}")

    tmp_lst = ['DNA', 'RNA', 'mRNA']
    for pos, value in enumerate(tmp_lst):
        print(f"value at pos {pos} is {value}")

    print(tmp_lst[0])

File: rust/src/intro.rs
fn _intro() {
    let x = 100.0;
    let mut y = 1.0;

    println!("x: {}, and y: {}", x, y);
    y = y * -0.3145;
    println!("x: {}, and y: {}", x, y);
    if y < x {
        println!("The difference is: {}", (x - y));
    } else if y == x {
        println!("The difference is: {}", (y - x));
    } else {
        println!("The difference is: {}", (y - x));
    }

    // let mut x = 5;
    // while x > 0 {
    //     println!("x is {}", x);
    //     x -= 1;
    // }

    let num_range = 3..7;
    for i in num_range {
        println!("i is {}", i);
    }

    let tmp_vec = vec!["DNA", "RNA", "mRNA"];
    for (pos, value) in tmp_vec.iter().enumerate() {
        println!("value at pos {} is {}", pos, value);
    }

    println!("{}", tmp_vec[0]);
}
